### ISSUE TITLE
route is registered out of nova

### ISSUE THREAD

**Issue by yurii-github** [KABBOUCHI/nova-logs-tool#9]
you do not check if current app is nova or not, you always registers routes like
nova-vendor/KABBOUCHI/logs-tool/log

**Comment by mstaack** [comment]
yeah this also adds a security risk! b/c routes are public.... and download of logs is possible without any auth!

**Comment by KABBOUCHI** [comment]
@mstaack routes is under `nova` middleware what risks you are talking about?

https://github.com/KABBOUCHI/nova-logs-tool/blob/7b59c497bac69ac33a438fa654d5c1a9c465d148/src/LogsToolServiceProvider.php#L45-L47

**Comment by mstaack** [comment]
i did a quick test and it seemed like downloading files is possible without nova login

**Comment by mstaack** [comment]
ok just did this again, sry for the misleading info. seems to work once canDownload() is used on the tool

**Comment by KABBOUCHI** [comment]
Yeah you should not use `true`, you should check the user permission. 

I'll modify the README file to prevent confusions

**Comment by mstaack** [comment]
Yeah makes sense! Thanks for the readme updates.
